### PR TITLE
Bitrunning Easy mode

### PR DESCRIPTION
**Pull request by OrbisAnima**
## About The Pull Request
- Doubles the amount of points received per domain won
- Makes sassysauge pizza available on the pizza delivery pool
- Makes Bitrunning disks size small, instead of normal

## How This Contributes To The Nova Sector Roleplay Experience
- 2x the points reduce the grind.
- More pizza selection is good.
- Bitrunning disks being small while allows for crazy bitrunning builds, also let people to just... have their disks in a belt or part of their inventory without having to balance their bitrunning and station life too much

## Proof of Testing
<details>
<summary>Screenshots/Videos</summary>

image

Currently available balance: 200 np

image

  
</details>

## Changelog
:cl:
balance: Bitrunning Domains now give double points
qol: Pizza Delivery for bitrunners now include the new Sausage pizza on their possible sponsors!
balance: Bitrunner disks are now small sized, from normal.
/:cl:

### CODE REVIEW

File: modular_nova/modules/bitrunning/code/flair.dm [hunk 1]
```diff
@@ -92,5 +92,6 @@
 		/obj/item/pizzabox/vegetable,
 		/obj/item/pizzabox/mushroom,
 		/obj/item/pizzabox/meat,
+		/obj/item/food/pizza/sassysage,
```
Comment (Stalkeros2) [inline modular_nova/modules/bitrunning/code/flair.dm:95]: This better be a pizza box obviously, otherwise it looks bad and can get moldy.

**Review by Stalkeros2 — COMMENTED** [review]
(Thank you GitHub mobile for turning singular comments into reviews.)

File: modular_nova/modules/bitrunning/code/flair.dm (hunk 1 above)
Comment (OrbisAnima) [reply modular_nova/modules/bitrunning/code/flair.dm:95]: it is

File: code/modules/bitrunning/server/loot.dm [hunk 2]
```diff
@@ -30,9 +30,9 @@
 
 	spark_at_location(cache) // abracadabra!
 	qdel(cache) // and it's gone!
-	SEND_SIGNAL(src, COMSIG_BITRUNNER_DOMAIN_COMPLETE, cache, generated_domain.reward_points)
+	SEND_SIGNAL(src, COMSIG_BITRUNNER_DOMAIN_COMPLETE, cache, generated_domain.reward_points*2) // NOVA EDIT CHANGE - Original : SEND_SIGNAL(src, COMSIG_BITRUNNER_DOMAIN_COMPLETE, cache, generated_domain.reward_points)
```
Comment (FlufflesTheDog) [inline code/modules/bitrunning/server/loot.dm:33]:
```suggestion
	SEND_SIGNAL(src, COMSIG_BITRUNNER_DOMAIN_COMPLETE, cache, generated_domain.reward_points * 2) // NOVA EDIT CHANGE - Original : SEND_SIGNAL(src, COMSIG_BITRUNNER_DOMAIN_COMPLETE, cache, generated_domain.reward_points)
```
gonna nitpick this

File: code/modules/bitrunning/server/loot.dm [hunk 3]
```diff
@@ -30,9 +30,9 @@
...
 
-	points += generated_domain.reward_points
+	points += (generated_domain.reward_points*2) // NOVA EDIT CHANGE - Original : points += generated_domain.reward_points
```
Comment (FlufflesTheDog) [inline code/modules/bitrunning/server/loot.dm:35]:
```suggestion
	points += (generated_domain.reward_points * 2) // NOVA EDIT CHANGE - Original : points += generated_domain.reward_points
```
same (I'm mad github won't let me do this in a single comment)

File: modular_nova/modules/bitrunning/code/domain_additions.dm [hunk 4]
```diff
@@ -0,0 +1,2 @@
+/datum/lazy_template/virtual_domain/gondola_asteroid
+	completion_loot = list(/obj/item/reagent_containers/pill/bitrunner_delight = 1) //This is just here as a patch over the current issue with gondola force ejecting you over transformation.
```
Comment (FlufflesTheDog) [inline modular_nova/modules/bitrunning/code/domain_additions.dm:2]: I don't think I'm a huge fan of this. There's a [fix](https://github.com/tgstation/tgstation/pull/88398) merged upstream now, just need to get that over here.

**Review by FlufflesTheDog — CHANGES_REQUESTED** [review]

**Review by FlufflesTheDog — APPROVED** [review]

**Pull request merged**